Tasks/Common/VstsAzureHelpers_/Utility.ps1

```diff
@@ -539,20 +540,18 @@ function Disconnect-AzureAndClearContext {
 
 function Disconnect-UsingAzModule {
     [CmdletBinding()]
-    param()
+    param(
+        [string]$restrictContext = 'False'
+    )
 
     if (Get-Command -Name "Disconnect-AzAccount" -ErrorAction "SilentlyContinue" -and CmdletHasMember -cmdlet Disconnect-AzAccount -memberName Scope) {	
+        if ($restrictContext -eq 'True') {
```

Comment:
Weren't we also going to use Clear-AzContext somewhere to do this ?

Reply:
Yes, earlier we were planning the same. But what I found was if have these command in order :
Disconnect-AzAccount -Scope Process
Clear-AzContext -Scope CurrentUser -Force
Clear-AzContext -Scope Process-Force,

Even then context was passing , URL : https://dev.azure.com/arjgupta22/p1/_releaseProgress?_a=release-environment-logs&releaseId=100&environmentId=100

The only way context is not passing when disconnect-azaccount -scope currentuser is run before clearing the context.